libs/util/nestjs/kafka/tsconfig.lib.json

```
@@ -7,5 +7,8 @@
     "types": ["node"]
   },
   "exclude": ["jest.config.ts", "**/*.spec.ts", "**/*.test.ts"],
-  "include": ["**/*.ts"]
+  "include": [
+    "**/*.ts",
+    "../../../../packages/amplication-server/src/graphql/graphqlSubscriptionKafka.service.ts"
+  ]
```

Absolutely an error. Good spot! 
```suggestion
  "include": ["**/*.ts"]
```